.github/workflows/analyze_comment.yml

```diff
@@ -10,8 +10,7 @@ jobs:
   comment:
     runs-on: ubuntu-latest
     if: >
-      ${{ github.event.workflow_run.event == 'pull_request' &&
-      github.event.workflow_run.conclusion == 'success' }}
+        github.event.workflow_run.workflows.event_name == 'pull_request' && github.event.workflow_run.workflows.conclusion == 'success'  
```

Comment:
This is the key about this problem.

We should use `github.event.workflow_run.workflows.event_name` instead of `github.event.workflow_run.event`.

And we shouldn't use the `${{}}`, I think.